gdnative-core/src/class.rs

```diff
@@ -301,15 +303,15 @@ impl<T: NativeClass> Instance<T> {
     }
 }
 
-impl<T> Clone for Instance<T>
+impl<T> RefCounted for Instance<T>
```

Comment:
Things that had `Clone` should still have `Clone`. `RefCounted` is meant to replace `Clone` **as a bound**, not wholesale.